analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.VisualBasic.nuspec

```diff
@@ -27,5 +27,6 @@
     <file src="bin\Release\Google.Protobuf.dll" target="analyzers" />
     <file src="tools-vbnet\*.ps1" target="tools\" />
     <file src="logos\sonarsource_64.png" target="images\" />
+    <file src="..\..\..\THIRD-PARTY-NOTICES.txt" target="license\" />
```

Comment:
I may be doing something very wrong here, but if I extract resulting vsix file with 7-zip it has logos directory, but there is no license folder inside.

Reply:
replied in https://github.com/SonarSource/sonar-dotnet/pull/4662#discussion_r669443997